lib/lapis_lazuli/generators/cucumber/template/Gemfile

```diff
@@ -5,4 +5,5 @@ source 'http://rubygems.org'
 # > bundle exec cucumber
 
 # LapisLazul itself
-gem 'lapis_lazuli', <%= config[:lapis_lazuli][:dependency] %>
+# gem 'lapis_lazuli', <%= config[:lapis_lazuli][:dependency] %>
+gem 'lapis_lazuli', :path => '../../../../../'
```

Comment:
Is this correct?

Reply:
No, not for production. It's for debugging so that it loads the gem from itself. Will update before pushing to master.